docs/reference/env-variables.rst

```diff
@@ -0,0 +1,703 @@
+      - ON, OFF
+      - Used to run the app as if it were run in rocprof. Forces command queue profiling on by default.
+    *
+      - HSAKMT_DEBUG_LEVEL
+      - 1, 2, ... 7
+      - When set to the highest level, the system will print memory allocation info.
+
+Environment variables effecting the rocclr backend of project clr.
+
+.. list-table:: Environment variables effecting the rocclr backend of project clr
+    :header-rows: 1
+    :name: clr-env-variables-rocclr-table
+
+    *
+      - Environment variable
+      - Values
+      - Description
+    *
+      - 
+      - 
+      - 
+
+rocclr environment variables
+----------------------------------
+
+AMD rocclr environment variables at release build:
+
+.. list-table:: AMD rocclr environment variables
+    :header-rows: 1
+    :name: rocclr-env-variables-release-table
+
+    *
+      - Environment variable
+      - Variable type
+      - Default type
+      - Description
+    *
+      - AMD_LOG_LEVEL
+      - int
+      - 0
+      - The default log level
+    *
+      - AMD_LOG_MASK
+      - uint
+      - 0X7FFFFFFF
+      - The mask to enable specific kinds of logs
+    *
+      - CQ_THREAD_STACK_SIZE
+      - size_t
+      - 256*Ki
+      - The default command queue thread stack size
+    *
+      - GPU_MAX_WORKGROUP_SIZE
+      - int
+      - 0
+      - Maximum number of workitems in a workgroup for GPU, 0 -use default
+    *
+      - ROC_HMM_FLAGS
+      - uint
+      - 0
+      - ROCm HMM configuration flags
+    *
+      - GPU_DEVICE_ORDINAL
+      - cstring
+      - 
+      - Select the device ordinal , comma separated list of available devices
+    *
+      - REMOTE_ALLOC
+      - bool
+      - FALSE
+      - Use remote memory for the global heap allocation
+    *
+      - GPU_CP_DMA_COPY_SIZE
+      - uint
+      - 1
+      - Set maximum size of CP DMA copy in KiB
+    *
+      - GPU_MAX_HEAP_SIZE
+      - uint
+      - 100
+      - Set maximum size of the GPU heap to % of board memory
+    *
+      - GPU_STAGING_BUFFER_SIZE
+      - uint
+      - 4
+      - Size of the GPU staging buffer in MiB
+    *
+      - GPU_DUMP_BLIT_KERNELS
+      - bool
+      - FALSE
+      - Dump the kernels for blit manager
+    *
+      - GPU_BLIT_ENGINE_TYPE
+      - uint
+      - 0x0
+      - Blit engine type: 0 - Default, 1 - Host, 2 - CAL, 3 - Kernel
+    *
+      - GPU_FLUSH_ON_EXECUTION
+      - bool
+      - FALSE
+      - Submit commands to HW on every operation. 0 - Disable, 1 - Enable
+    *
+      - CL_KHR_FP64
+      - bool
+      - TRUE
+      - Enable/Disable support for double precision
+    *
+      - AMD_OCL_BUILD_OPTIONS
+      - cstring
+      - 0
+      - Set clBuildProgram,  and clCompileProgram, 's options , override
+    *
+      - AMD_OCL_BUILD_OPTIONS_APPEND
+      - cstring
+      - 0
+      - Append clBuildProgram,  and clCompileProgram, 's options
+    *
+      - AMD_OCL_LINK_OPTIONS
+      - cstring
+      - 0
+      - Set clLinkProgram, 's options , override
```

Comment:
Again, the "override" is not specified very well
```suggestion
      - Set clLinkProgram's options, override
```

Reply:
It should be something different. The 's possessive is grammatically incorrect.